doc-content/enterprise-only/getting-started/data-model-proc.adoc

```diff
@@ -1,51 +1,72 @@
 [id='_defining_a_data_model']
-= Viewing the *Mortgage_Process* data objects
+= Creating the mortgage-process data objects
 
 Data objects are the building blocks for the rule assets that you create. Data objects are custom data types implemented as Java classes in specified packages of your project. These custom data types determine what data your assets and your decision services are based on.
 
-This tutorial uses the following data objects:
+The *mortgage-process* project uses the following data objects:
 
 * `Applicant`
-* `Application`
 * `Property`
 * `ValidationErrorDO`
+* `Application`
 
-== Viewing the Applicant data object
-Follow these steps to familiarize yourself with the predefined *Applicant* data object, which contains information about the applicant. This is the basic information required to apply for the loan in this tutorial.
+== Creating the Applicant data object
+Follow these steps to create the *Applicant* data object, which contains information about the applicant. This is the basic information required to apply for the loan in this tutorial.
 
 .Procedure
-. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *Mortgage_Process*.
-. Enter `Applicant.java` in the project's asset search box, press Enter, and click the *Applicant* data object.
-. Review the *Applicant* data object values.
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*, and click *mortgage-process*.
```

Comment:
*Mortgage_Process* search for

Reply:
This is correct as we are telling them to create a new project with a name change from the sample project, which we also reference as a resource.